target/linux/generic/hack-6.6/765-mxl-gpy-control-LED-reg-from-DT.patch

```diff
@@ -90,16 +100,4 @@ Signed-off-by: David Bauer <mail@david-bauer.net>
 +
  static int gpy_config_init(struct phy_device *phydev)
  {
- 	int ret;
-@@ -271,7 +307,10 @@ static int gpy_config_init(struct phy_de
- 
- 	/* Clear all pending interrupts */
- 	ret = phy_read(phydev, PHY_ISTAT);
--	return ret < 0 ? ret : 0;
-+	if (ret < 0)
-+		return ret;
-+
-+	return gpy_led_write(phydev);
```

Comment:
This still belongs in gpy_config_init

Reply:
@graysky2 
Change to the following format:
````
@@ -216,7 +223,10 @@ static int gpy_ack_interrupt(struct phy_

 	/* Clear all pending interrupts */
 	ret = phy_read(phydev, PHY_ISTAT);
-	return ret < 0 ? ret : 0;
+	if (ret < 0)
+		return ret;
 }
````
。。。。。。
````
 static int gpy_config_init(struct phy_device *phydev)
 {
 -	/* Nothing to configure. Configuration Requirement Placeholder */
 - 		return 0
 +    return gpy_led_write(phydev);
}